CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 8.0.0-alpha.1
+
+- The first public preview of our [new high-performance core API and extensions](https://github.com/App-vNext/Polly/issues/1048) - Thanks to:
```

Comment:
Can we also reference the samples directory (to be merged) and this readme that explains the architecture?

https://github.com/App-vNext/Polly/blob/main/src/Polly.Core/README.md

Reply:
I'll add the samples once it's merged (or you can rebase onto this once merged and add it yourself).